indico/modules/events/registration/templates/display/regform_display.html

```diff
@@ -26,6 +26,13 @@
 
 
 {% block content %}
+    {% if not event.can_access(session.user) %}
```

Comment:
I guess there is no other situation in which `.can_access` returns `False`, since otherwise this template wouldn't even be rendered? I was wondering whether we should check for `.has_regform_in_acl` anyway, or just add a comment explaining that...

Reply:
comment sounds reasonable - and you are right, there's no other way to get there if you do not have access